modules/server/src/main/ts/bedrock/compiler/Imports.ts

```diff
@@ -85,10 +100,14 @@ var addTest = function (testFilePath) {
     console.error('no test list to add tests to');
   }
 };
+var handleParseError = function (testFilePath, error) {
```

Comment:
This looks almost identical to the above. Can the string be shared?

Reply:
It's not identical at all, one is `JS` and the other is strict `TS`, so no they can't be shared sorry.

Reply:
Ah. I see.

Reply:
What benefit do you get from making the TS one TS? Can it pick up any errors at that point? Would you be able to share it if they were both JS? Probably a stupid question, I'm just genuinely curious.

Reply:
Or is the TS one required because there is a strict setting on?

Reply:
The TS one is required as bedrock reads from the projects tsconfig, so if strict is on it'll complain and fail to compile.